site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -267,7 +266,7 @@ the keyword.
 ```
 
 When the user types "nt" into the omnibox, it activates the extension. To signal this to the user,
-it greyscales the provided 16x16 icon and includes it in the omnibox next to the extension name.
+it grayscales the provided 16x16 icon and includes it in the omnibox next to the extension name.
```

Comment:
A controversial change for the ages. (All good, I don't mind if it gets changed)

Reply:
:)